lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -9,7 +9,7 @@
             'js/student_account/views/FormView',
             'text!../../../templates/financial-assistance/financial_assessment_form.underscore',
             'text!../../../templates/financial-assistance/financial_assessment_submitted.underscore',
-            'text!templates/student_account/form_field.underscore',
+            'text!lms/djangoapps/student_account/templates/student_account/form_field.underscore',
```

Comment:
Ok, but those files are also used by regular mako templates.  Do I need to update the template processors to look for static/X/templates as well?